shell/platform/android/image_external_texture_vk.h

```diff
@@ -30,9 +35,21 @@ class ImageExternalTextureVK : public ImageExternalTexture {
   void ProcessFrame(PaintContext& context, const SkRect& bounds) override;
   void Detach() override;
 
+  sk_sp<flutter::DlImage> FindImage(uint64_t key);
+
+  void UpdateKey(uint64_t key);
+
+  void AddImage(sk_sp<flutter::DlImage> image, uint64_t key);
+
   const std::shared_ptr<impeller::ContextVK> impeller_context_;
 
   fml::jni::ScopedJavaGlobalRef<jobject> android_image_;
+
+  std::array<std::pair<uint64_t, sk_sp<flutter::DlImage>>,
+             kImageReaderSwapchainSize>
+      images_ = {std::make_pair(0, nullptr), std::make_pair(0, nullptr),
+                 std::make_pair(0, nullptr)};
+  std::array<uint64_t, kImageReaderSwapchainSize> keys_ = {0, 0, 0};
```

Comment:
add comment explaining how keys_ is ordered.

Reply:
also, do we need to split this into two different arrays? Like what if we just kept images_ ordered?

Reply:
No we don't really need two, I should just combine the arrays.